src/main/java/com/mapr/synth/Synth.java

```diff
@@ -97,6 +102,20 @@ public static void main(String[] args) throws IOException, CmdLineException, Int
         final SchemaSampler sampler = new SchemaSampler(opts.schema);
         final AtomicLong rowCount = new AtomicLong();
 
+        Template template = null;
+        if (opts.format == Format.TEMPLATE) {
+            final Configuration cfg = new Configuration(Configuration.VERSION_2_3_21);
+            cfg.setDefaultEncoding("UTF-8");
+            cfg.setTemplateExceptionHandler(TemplateExceptionHandler.RETHROW_HANDLER);
+            cfg.setDirectoryForTemplateLoading(opts.template.getAbsoluteFile().getParentFile());
+
+            try {
+                template = cfg.getTemplate(opts.template.getName());
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
This is never the right answer.  An exception here is non-recoverable and we should just not catch the exception.